src/styles/styles.css

```diff
@@ -0,0 +1,306 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700;800;900&display=swap');
+
+body {
+  width: 100%;
+  height: auto;
+  background-color: #f5f6fa;
+  text-decoration: none;
+}
+
+p {
+  margin-top: 0.4rem;
+  margin-bottom: 0;
+}
+
+#root {
+  width: 85%;
+  height: 70%;
+  margin: auto auto;
+  margin-top: 5%;
+  padding-bottom: 30rem;
+  border-radius: 4px;
+  box-shadow: 40px 60px 60px 0 rgba(0, 0, 0, 0.2);
+  background-color: #fafafa;
+}
+
+.navbar {
+  width: 100%;
+  height: 5.938rem;
+  background-color: #ffff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.Bookstore-CMS,
+.categories-page {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  padding-left: 1rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.menuitems {
+  width: 20%;
+}
+
+.BOOKS {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.CATEGORIES {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navbar-icon {
+  height: 60%;
+  border-radius: 50%;
+  margin-left: auto;
+  margin-right: 2rem;
+  justify-self: flex-end;
+  cursor: pointer;
+  border: solid 2px #e8e8e8;
+}
+
+.book-card {
+  width: 90%;
+  margin: auto;
+  margin-top: 1.5rem;
+  border-radius: 4px;
+  padding: 1rem 1rem;
+  height: 7rem;
+  border: solid 1px #e8e8e8;
+  background-color: #ffff;
+}
+
+.Line {
+  width: 100%;
+  height: 0.002rem;
+  margin: 2.5rem 0 1.813rem 0;
+  border: solid 1px #dad9d9;
+}
+
+.addBookHeader {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin-left: 2.5rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+#title-input,
+#author-input {
+  background: #ffff;
+  border: solid 1px #e8e8e8;
+  width: 35%;
+  height: 2.8rem;
+  font-size: 1rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  margin-left: 2rem;
+  margin-top: 2rem;
+}
```

Comment:
- [ ] Kindly give some padding around `BOOK TITLE` and `Book Author` to match the given design.

- [ ] You missed a `down arrow` in the `Book Author` input tag. Kindly fix this issue also to match the given design.

![image](https://github.com/demesameneshoa/book-store/assets/60083437/0e201c18-dfc1-4968-8792-7820462583dc)